tests/template_tests/syntax_tests/test_querystring.py

```diff
@@ -6,72 +6,79 @@
 
 
 class QueryStringTagTests(SimpleTestCase):
-    def setUp(self):
-        self.request_factory = RequestFactory()
-
-    @setup({"querystring_empty": "{% querystring %}"})
-    def test_querystring_empty(self):
-        request = self.request_factory.get("/")
-        template = self.engine.get_template("querystring_empty")
-        context = RequestContext(request)
+
```

Comment:
```suggestion
```
(nit)

Reply:
Thank you, I wonder why this wasn't caught by our lint tools! :thinking: